torch/_meta_registrations.py

```diff
@@ -7,11 +7,33 @@
     elementwise_dtypes,
 )
 from torch._prims_common.wrappers import out_wrapper
+from torch.utils._pytree import tree_map
 
 from typing import List, Optional
 
+aten = torch.ops.aten
+
 meta_lib = torch.library.Library("aten", "IMPL", "Meta")
 
+meta_table = {}
+
+def register_meta(op, register_dispatcher=True):
```

Comment:
What's a case when someone would want to set `register_dispatcher` to `False`?

Reply:
If there's already a meta function registered to that operator, for example, but we want a separate one in Python.